### PR TITLE
app-i18n/uim: libressl support

### PR DESCRIPTION
**Pull request by doughdemon**
@i18n @blueness

### CODE REVIEW

**Comment by naota** [comment]
@gentoo/cjk

File: app-i18n/uim/uim-1.8.6-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+	m17n-lib? ( >=dev-libs/m17n-lib-1.3.1 )
+	ncurses? ( sys-libs/ncurses:0= )
+	nls? ( virtual/libintl )
+	qt4? ( dev-qt/qtgui:4[qt3support] )
+	skk? ( app-i18n/skk-jisyo )
+	sqlite? ( dev-db/sqlite:3 )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl:0 )
+	)
```
Comment (blueness) [inline app-i18n/uim/uim-1.8.6-r2.ebuild]: We should be using dev-libs/openssl:0= and dev-libs/libressl:0= in anticipation of future slot/subslot changes.  Both packages are sensitive to ABI changes as they begin to deprecate weak session/presentations like SSLv2 and SSLv3.

**Comment by blueness** [comment]
I don't know the package, but the libressl bits look okay.

File: app-i18n/uim/uim-1.8.6-r2.ebuild (hunk 1 above)
Comment (doughdemon) [inline app-i18n/uim/uim-1.8.6-r2.ebuild]: > We should be using dev-libs/openssl:0= and dev-libs/libressl:0= [...]

fixed

**Comment by blueness** [comment]
Okay I pushed this.

@gentoo/cjk If I overstep my bounds, please revert.  But as I said, this looks like pretty straight forward stuff for libressl.

**Pull request closed without merging**